src/index.html

```diff
@@ -11,8 +11,54 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+
+      <p class="card__apple">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__product">Product code: 195434</p>
+
+      <div class="stars stars--4">
+        <div class="stars__image"></div>
+        <div class="stars__image"></div>
+        <div class="stars__image"></div>
+        <div class="stars__image"></div>
+        <div class="stars__image"></div>
+        <div class="stars__text">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price_text">Price:</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
        <p class="card__price_text">Price:</p>
```